source/scripting_v3/GTA/World.cs

```diff
@@ -702,29 +741,30 @@ public static Vehicle CreateVehicle(Model model, Vector3 position, float heading
 			return new Vehicle(Function.Call<int>(Hash.CREATE_VEHICLE, model.Hash, position.X, position.Y, position.Z, heading, false, false));
 		}
 		/// <summary>
-		/// Spawns a <see cref="Vehicle"/> of a random <see cref="Model"/> at the position specified. Does not work currently.
+		/// Spawns a <see cref="Vehicle"/> of a random <see cref="Model"/> at the position specified.
 		/// </summary>
 		/// <param name="position">The position to spawn the <see cref="Vehicle"/> at.</param>
 		/// <param name="heading">The heading of the <see cref="Vehicle"/>.</param>
+		/// <param name="predicate">The method that determines whether a model should be considered when choosing a random model for the <see cref="Vehicle"/>.</param>
 		/// <remarks>returns <c>null</c> if the <see cref="Vehicle"/> could not be spawned.</remarks>
-		public static Vehicle CreateRandomVehicle(Vector3 position, float heading = 0f)
+		public static Vehicle CreateRandomVehicle(Vector3 position, float heading = 0f, Func<Model, bool> predicate = null)
```

Comment:
Bit worried this will break some existing script that tries to use this method. Then again, this didn't even work before, so probably doesn't matter.